src/lib/device-types/index.ts

```diff
@@ -16,27 +17,14 @@ import {
 	getLogoUrl,
 } from './build-info-facade';
 import { getImageKey, IMAGE_STORAGE_PREFIX, listFolders } from './storage';
-
-const { api } = sbvrUtils;
-const { BadRequestError, InternalRequestError, NotFoundError } = errors;
-export type { BadRequestError, NotFoundError };
+import {
+	UnknownDeviceTypeError,
+	InvalidDeviceTypeError,
+	UnknownVersionError,
+} from '../errors';
```

Comment:
Why did you move these errors out of here? They're quite specific to device types and here (or a in new file `lib/device-types/`) is their place IMO.

Reply:
@dfunckt the same can be said about all errors that are in `errors.ts`, since they are also quite specific (`InaccessibleAppError`, `NoDevicesFoundError`, etc.). The reason I think it's better if they are in errors is that even though the error is for a device type, it can be used in different contexts (when creating an app, when provisioning a device, etc.), and we should either have all error definitions in one place, or have them all defined in the resources they act on (app, device, etc.). Currently, it is a mix of the two, and this change just makes everything consistent. Does that make sense?

Reply:
Gotcha, that makes sense to me -- agreed.